web/models/user.ex

```diff
@@ -40,4 +54,17 @@ defmodule CodeCorps.User do
         changeset
     end
   end
+
+  defp prefix_url(changeset, key) do
+    changeset
+    |> update_change(key, &do_prefix_url/1)
+  end
+
+  defp do_prefix_url(nil), do: nil
+
+  defp do_prefix_url("http://" <> rest), do: "http://" <> rest
+
+  defp do_prefix_url("https://" <> rest), do: "https://" <> rest
+
+  defp do_prefix_url(value), do: "http://" <> value
```

Comment:
Related functions should be grouped together without blank lines: https://github.com/rrrene/elixir-style-guide